vm/state_manager/src/errors.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use amt::Error as AmtErr;
+use std::fmt;
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
```

Comment:
If this type is just forwarding errors, can just change the Error variant to string and get rid of this. Only preference for this though, can keep if you'd like